crates/common/tedge_config/src/system_services/log_config.rs

```diff
@@ -19,14 +19,23 @@ pub fn get_log_level(
     }
 }
 
-/// Initialize a `tracing_subscriber`
-/// Reports all the log events sent either with the `log` crate or the `tracing` crate.
+/// Initializes a tracing subscriber with a given log level if environment
```

Comment:
```suggestion
/// Initializes a tracing subscriber with a given log level through --debug, if the environment
```

Reply:
It's the caller who chooses whether or not to call this function when `--debug` is present or not, and the function doesn't check if `--debug` is present or not, so the function doc should not mention it.